.environment/eslint/run-eslintCheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+function note() {
+    echo "eslint-check> info: ${*}"
+}
+
+function js_fmt_check() {
+    note "Checking JavaScript formatting."
+    MODIFIED_TSX_FILES_COUNT=$(git status --porcelain | grep "\.tsx$" | wc -l)
+    MODIFIED_JS_FILES_COUNT=$(git status --porcelain | grep "\.js$" | wc -l)
+    if [[ ${MODIFIED_TSX_FILES_COUNT?} != 0 ]] || [[ ${MODIFIED_JS_FILES_COUNT?} != 0 ]]; then
```

Comment:
can you remove the double-brackets here, or change the comparison to `-ne 0`? (see https://github.com/CDCgov/prime-reportstream/pull/3064 for where we had to fix this elsewhere)